packages/ndla-ui/src/Article/Article.tsx

```diff
@@ -217,8 +197,10 @@ export const ArticleTitle = ({
   return (
     <ArticleHeader padded>
       <ArticleHGroup>
-        {!!contentType && <ContentTypeBadgeNew contentType={contentType} />}
-        {!!heartButton && <ArticleActionWrapper>{heartButton}</ArticleActionWrapper>}
+        <HStack justify="space-between">
+          {!!contentType && <ContentTypeBadgeNew contentType={contentType} />}
+          {!!heartButton && heartButton}
+        </HStack>
```

Comment:
Er den lengste innholdstypen vår lang nok til å treffe hjerteknappen?

Reply:
Den så fin ut når jeg testa lokalt på to-tre

Reply:
Legg på en liten gap just in case